tests/src/performance/Scenario/JitBench/JitBench.csproj

```diff
@@ -1,58 +1,52 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
.NET Core csproj! I like it! :)
Ideally, all projects should move to it!

Reply:
Cool, I was worried this might be a controversial move ; )